### PR TITLE
Update doc strings for cube_build

### PR DESCRIPTION
**Pull request by jemorrison**
Add the doc string to all modules. Also did a general clean up of comments in the code to make it
follow python rules.

### CODE REVIEW

File: docs/jwst/cube_build/index.rst [hunk 1]
```diff
@@ -10,4 +10,12 @@ Cube Building
    examples.rst
    reference_files.rst
    
-.. automodapi:: jwst.cube_build
+.. automodapi:: jwst.cube_build.cube_build_step
+.. automodapi:: jwst.cube_build.data_types
```
Comment (jdavies-st) [inline docs/jwst/cube_build/index.rst]: Do we want all these lower level functions in the main docs?  I would suspect all we need is `cube_build_step` for the docs.  Or do we expect users to need to call these lower level functions?

File: jwst/cube_build/coord.py [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-    CRVAL1,CRVAL2 and the tangent point
+    """ Compute the tangent projection coordinates (xi,eta) from ra,dec
+    using crval1 and crval2 (the tangent point).
 
-    Input Values
+    Parameters
     ____________
-    CRVAL1: ra value of tangent point
-    CRVAL2: dec value of tangent point
+    crval1: ra value of tangent point
```
Comment (jdavies-st) [inline jwst/cube_build/coord.py]: Here, it should indicate the type, so
```
crval1 : float
    RA value of tangent point
```
Sae for `crval2`.

File: jwst/cube_build/cube_build_wcs_util.py [hunk 3]
```diff
@@ -146,17 +142,19 @@ def find_footprint_NIRSPEC(input, flag_data, coord_system):
...
-    Test for 0/360 wrapping in ra values. If exists it makes it difficult to determine
+    """Test for 0/360 wrapping in ra values.
+
+    If exists it makes it difficult to determine
     ra range of IFU cube. So put them all on "one side" of 0/360 border
 
     Input
     -----
-    ravalues a numpy array of ra values
+    ravalues : a numpy array
```
Comment (jdavies-st) [inline jwst/cube_build/cube_build_wcs_util.py]:
```
ravalues : numpy.ndarray
    RA values
```

File: docs/jwst/cube_build/index.rst (hunk 1 above)
Comment (jemorrison) [reply docs/jwst/cube_build/index.rst]: I was not sure if every module had to be listed in index.rst or just the main code.
I will make it just cube_build_step

**Comment by jemorrison** [comment]
@jdavies-st I am not sure why there was a problem with the jenkins testing.

File: jwst/cube_build/coord.py [hunk 4]
```diff
@@ -62,11 +66,13 @@ def std2radec(crval1, crval2, xi, eta):
     Parameters
     ____________
     crval1 : float
-      ra value of tangent point
+      RA value of tangent point
     crval2 : float
-      dec value of tangent point
-    xi, eta : float, float
-      rectangular coordinates of tangent plane projected ra,dec
+      RA value of tangent point
```
Comment (jdavies-st) [inline jwst/cube_build/coord.py]: `DEC value` here I believe.

**Comment by jdavies-st** [comment]
Oh, it's a problem with the `Jenkinsfile` not being pinned to `numpy 1.15.2` in your branch.  Not a problem.

**Comment by jdavies-st** [comment]
Jenkins failure is unrelated to this PR.  A rebase to `jwst/master` and a force push should fix it.

**Pull request merged**